cat/CaT-service.yaml

```diff
@@ -3802,6 +3924,26 @@ components:
               - buyer
               - supplier
             example: supplier
+            
+    DocumentAudienceType:
+      description: Intended audience for a document
+      type: string
+      enum:
+        - buyer
+        - seller
```

Comment:
Really minor point - we have `buyer|supplier` being returned from some other endpoints, should probably standardise on one or t'other